hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -302,67 +302,19 @@ N.B `flashrom` does not support clearing of the write-protect regions and will f
-
-### Using Raspberry Pi Imager to update the bootloader (recommended)
-Raspberry Pi Imager provides a GUI for updating the bootloader
-
-1. Download [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/)
-2. Select a spare SD card. The contents will get overwritten!
-3. Launch `Raspberry Pi Imager`
-4. Select `Misc utility images` under `Operating System`
-5. Select `Bootloader`
-6. Select a boot-mode 
-7. Select `SD card` and then `Write`
-
-### Using raspi-config to update the bootloader from within Raspberry Pi OS
-To change the boot-mode or bootloader version from within Raspberry Pi OS run [raspi-config](../../configuration/raspi-config.md)
-
-1. Run `sudo raspi-config`
-2. Select `Advanced Options`
-3. Select `Boot Order`
-4. Select SD, USB or Network boot
-5. Reboot
-
-### Custom boot-order
-The `rpi-eeprom-config` utility has an editor mode which can be used to make quick changes to the EEPROM config file. Use this to define the `BOOT_ORDER` setting directly
-
-```
-sudo apt update
-sudo apt full-upgrade
-sudo -E rpi-eeprom-config --edit
-```
-Make changes, then save and quit. `rpi-eeprom-config` will print details of the EEPROM release and how to revert changes. Reboot to apply the bootloader update.
-
-The [Boot EEPROM](booteeprom.md) page provides more information about the EEPROM flashing process.
-
-
-#### Hardware compatibility for USB mass storage devices
-Before attempting to boot from a USB mass storage device it is advisible to verify that the device works correctly under Linux. Boot using an SD card and plug in the USB mass storage device. This should appears as a removable drive.
-
-*Spinning hard-disk drives nearly always require a powered USB hub. Even if it appears to work you are likely to encounter intermittent failures without a powered USB HUB*
-
-This is especially important with USB SATA adapters which may be supported by the bootloader in mass storage mode but fail if Linux selects [USB Attached SCSI - UAS](https://en.wikipedia.org/wiki/USB_Attached_SCSI) mode.
-
-See this [forum thread](https://www.raspberrypi.org/forums/viewtopic.php?t=245931) about UAS and how to add [usb-storage.quirks](https://www.kernel.org/doc/html/v5.0/admin-guide/kernel-parameters.html) to workaround this issue.
-
-#### Multiple bootable drives
-When searching for a bootable partition the bootloader scans all USB mass storage devices in parallel and will select the first to respond. If the boot partition does not contain a suitable start.elf file the next available device is selected.
+### config.txt section
+From Raspberry Pi OS 2021-11-01, the firmware now reads the bootloader configuration from the EEPROM. If a section called `[config.txt]` is found then the contents from the start of this section to the end of the file is appended in memory, to the contents of the `config.txt` file read from the boot partition.  This can be used automatically apply over-clocking or dtoverlay settings to every operating system.  
```

Comment:
I **suspect** the date here is wrong... :laughing: 

"used automatically apply" -> "used to automatically apply"

Reply:
done